apprise/plugins/__init__.py

```diff
@@ -115,9 +115,6 @@ def __load_matrix(path=abspath(dirname(__file__)), name='apprise.plugins'):
         # Add our module name to our __all__
         __all__.append(plugin_name)
 
-        # Load our module into memory so it's accessible to all
-        globals()[plugin_name] = plugin
```

Comment:
Just want to backcheck, are you sure this isn't required?

My understanding is all modules imported today using the `import` are inserted into `globals()`.  You can see yourself:
```python
# Python 3.10.6 (main, Aug  2 2022, 00:00:00) [GCC 11.3.1 20220421 (Red Hat 11.3.1-2)] on linux
# Type "help", "copyright", "credits" or "license" for more information.
>>> globals()
{'__name__': '__main__', '__doc__': None, '__package__': None, '__loader__': <class '_frozen_importlib.BuiltinImporter'>, '__spec__': None, '__annotations__': {}, '__builtins__': <module 'builtins' (built-in)>}
>>> import re
>>> globals()
{'__name__': '__main__', '__doc__': None, '__package__': None, '__loader__': <class '_frozen_importlib.BuiltinImporter'>, '__spec__': None, '__annotations__': {}, '__builtins__': <module 'builtins' (built-in)>, 're': <module 're' from '/usr/lib64/python3.10/re.py'>}
```

The takeaway is that the library is added into the environment.  While doing this manually (As Apprise does); this is effectively mimicking this.  It also allows subsequent libraries to not re-use unnecessary io to detect the module wasn't loaded already and load it again.

HOWEVER, i could be wrong here. But i'm tempted to ask you to role this back just because i'm not sure if it's really a bad thing or not. Can you confirm if your other subsequent code changes fail without this change? Is that why it was made? (just curious)

Reply:
> Just want to backcheck, are you sure this isn't required?

I am pretty much sure, because I've identified this spot as the root of all evil. All the other changes within this patch kind of pivot around eradicating it.

> My understanding is all modules imported today using the import are inserted into `globals()`.

That is correct. The problem here is that `plugin` is not the reference to the module, but a reference to the class. This messes all up, and will make Python 3.11 croak, as you can see at https://github.com/caronc/apprise/pull/681#issuecomment-1272195760 and https://github.com/caronc/apprise/pull/681#pullrequestreview-1135169123. I listed likely reasons for that, regarding optimizations within the interpreter, in my original post.

Reply:
Thanks for just refreshing my brain on this!